aspnetcore-webapi/WebApiReturnHttp500/Tests/Usings.cs

```diff
@@ -0,0 +1,8 @@
+// Global using directives
```

Comment:
No need for this comment, we already know that these are global using directives because each line starts with `global using`, so this comment can be removed 😄